src/YT/YT_Inline.cpp

```diff
@@ -259,15 +242,14 @@ void YT_Inline()
 #  endif // #ifdef PARTICLE
 
 // 4. prepare local patches for libyt
-   YT_AddLocalGrid( GID_LvStart, NPatchAllRank, NField, FieldList);
+   YT_AddLocalGrid( NField, FieldList, pc);
 
 // 5. perform yt inline analysis
-   if ( yt_inline_argument( "yt_inline_inputArg", 1, "\'Dens\'" ) != YT_SUCCESS )    Aux_Error( ERROR_INFO, "yt_inline_inputArg() failed !!\n" );
```

Comment:
@koarakawaii There is this line of code that I changed because it gave errors with my version of libyt, but probably that was because my version was too old (?). I will change it back and push it again.

Reply:
@koarakawaii On a similar note, I also had to rename the field in the yt_inline.py plotting script. I hope that everything is consistent now. Could you please check again? I pushed the version where I changed the name of the field back to "Dens".  Thank you

Reply:
@KunkelAlexander: Thank you for the help! After changing this, the test can be run with this PR :grinning: !

Reply:
@koarakawaii  Thank you so much for your help :) I guess I am happy then. ;)